src/modules/peek/Peek.FilePreviewer/Controls/UnsupportedFilePreview/InformationalPreviewControl.xaml

```diff
@@ -32,10 +32,22 @@
                 FontSize="26"
                 FontWeight="SemiBold"
```

Comment:
Could this also be `TitleTextBlockStyle`?

Reply:
If you like a different UI layout please create an issue where we can discuss it. I won't implement any layout changes in this PR. This is iut of scoobe.